tck.sh

```diff
@@ -34,49 +34,53 @@ docker build -t $PYTHON_TCK_NAME ./
 docker network create $NETWORK_NAME
 
 #  primary tck tests for shopping cart
+docker run -d --network $NETWORK_NAME --name $USER_FUNCTION_NAME -p 8080:8080 $PYTHON_TCK_NAME server shoppingcart
```

Comment:
the tck.sh script did not work for me and fails with:

```
Step 7/16 : COPY ./protobuf /python-support/protobuf
COPY failed: stat /var/lib/docker/tmp/docker-builder373354787/protobuf: no such file or directory
```

an with that the script goes on and fails to run the TCK:
```
Successfully built grpcio-reflection
Installing collected packages: attrs, appier, google-api, six, protobuf, googleapis-common-protos, grpcio, grpcio-tools, pyparsing, packaging, toml, pluggy, py, iniconfig, pytest, grpcio-reflection
Successfully installed appier-1.20.5 attrs-20.2.0 google-api-0.1.12 googleapis-common-protos-1.52.0 grpcio-1.33.2 grpcio-reflection-1.33.2 grpcio-tools-1.33.2 iniconfig-1.1.1 packaging-20.4 pluggy-0.13.1 protobuf-3.13.0 py-1.9.0 pyparsing-2.4.7 pytest-6.1.2 six-1.15.0 toml-0.10.1
WARNING: You are using pip version 19.3.1; however, version 20.2.4 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Removing intermediate container f8009cb6449b
 ---> 116fc768096d
Step 6/16 : COPY ./scripts /python-support/scripts
 ---> d04fec48bf4a
Step 7/16 : COPY ./protobuf /python-support/protobuf
COPY failed: stat /var/lib/docker/tmp/docker-builder373354787/protobuf: no such file or directory
+ docker network create tck-network-IXI8N1IFFE
6ce402e9380542dfee68194465b263d20c52620d5209d9a1177e207ac640b309
+ docker run -d --network tck-network-IXI8N1IFFE --name cloudstate-function-IXI8N1IFFE -p 8080:8080 cloudstate-python-tck-dev:IXI8N1IFFE server shoppingcart
Unable to find image 'cloudstate-python-tck-dev:IXI8N1IFFE' locally
docker: Error response from daemon: pull access denied for cloudstate-python-tck-dev, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
```

Reply:
We might to set 
```
set -o errexit
```
for this script, as it would exit immediately if any statement returns a non-true return value.

Reply:
added